benches/README.md

```diff
@@ -44,6 +44,30 @@ Notes:
 - On Graviton 3 and 4, RPO256 and RPX256 are run with SVE acceleration enabled.
 - On AMD EPYC 9R14, RPO256 and RPX256 are run with AVX2 acceleration enabled.
 
+### Data Structure Benchmarks
```

Comment:
I would probably update the sections of this file to put `Smt` benchmarks into a higher level section. For example:

- Benchmarks
  - Hash functions
    - Scenario 1
    - Scenario 2
  - Sparse Merkle tree
    - Scenario 1
    - Scenario 2